src/selectedtests/app/wsgi.py

```diff
@@ -0,0 +1,3 @@
+from selectedtests.app.app import main
```

Comment:
I moved `wsgi.py` out of `src/` 👍 

I don't want to simplify app.py the way you suggest, it is easier to test as-is, plus if we want to avoid dependency injection later on, then `app` can be passed as an argument to `create_app`, rather than being defined on the file level.

Added an entry point 👍

Reply:
Actually, since `wsgi.py` is used by gunicorn, and gunicorn is required in setup.py, I think it's fair to include `wsgi.py` in `src/` so I'm adding it back.